config/resources/dossier-domain.lisp

```diff
@@ -14,7 +14,6 @@
                                 :inverse t
                                 :as "publication-flows")
               (sign-flow        :via      ,(s-prefix "sign:behandeltDossier")
-                                :inverse t
```

Comment:
Waarom deze inverse weg?
Als dit juist is, dan moet de inverse in de andere richting liggen

https://github.com/kanselarij-vlaanderen/kaleidos-project/blob/26269de71ffc45d4e5aa158e59dfddd4886f0909/config/resources/handtekening-domain.json#L46

Reply:
Ik vermoed eerder een foutje?
De relatie lag wel juist.
"Een handtekenAangelegenheid behandelt een dossier"
![image](https://user-images.githubusercontent.com/22245223/134539262-f8703452-fba5-4e97-b3b2-b83fe521b47e.png)